arch/riscv/core/fatal.c

```diff
@@ -32,6 +32,14 @@ FUNC_NORETURN void z_riscv_fatal_error(unsigned int reason,
 				       const z_arch_esf_t *esf)
 {
 	if (esf != NULL) {
+#if defined(CONFIG_RISCV_ISA_RV32E)
+		LOG_ERR("     a0: " PR_REG "    t0: " PR_REG, esf->a0, esf->t0);
+		LOG_ERR("     a1: " PR_REG "    t1: " PR_REG, esf->a1, esf->t1);
+		LOG_ERR("     a2: " PR_REG "    t2: " PR_REG, esf->a2, esf->t2);
```

Comment:
`LOG_ERR` for a0..2 seems to be the same with and without the embedded. Can you move it before `#if defined(CONFIG_RISCV_ISA_RV32E)`?

Reply:
done